exporter/tanzuobservabilityexporter/transformer.go

```diff
@@ -73,6 +74,18 @@ func (t *traceTransformer) Span(orig pdata.Span) (span, error) {
 
 	tags[labelSpanKind] = spanKind(orig)
 
+	if droppedEventsCount := orig.DroppedEventsCount(); droppedEventsCount > 0 {
+		tags[labelDroppedEventsCount] = strconv.Itoa(int(droppedEventsCount))
```

Comment:
Nit pick: On 32 bit machines this won't work if droppedEventsCount>= 2^31.  Consider using strconv.FormatUint() instead.

Reply:
done